index.js

```diff
@@ -76,21 +74,22 @@ module.exports = function(source) {
 
             this.addDependency(entityPath);
 
-            return vowFs
-                .exists(entityPath)
-                .then(exist => {
-                    // BemFile
-                    return {
-                        cell : bemCell,
-                        exist,
-                        // prepare path for require cause relative returns us string that we couldn't require
-                        path : unifyPath(requiredPath(path.relative(path.dirname(this.resourcePath), entityPath)))
-                    };
-                });
+            return new Promise(resolve => this.fs.stat(entityPath, err => {
+                return resolve(!err);
+            }))
+            .then(exist => {
```

Comment:
Each `then` produces new Promise object.

Reply:
It was minimum diff change
I fixed it